staging/src/k8s.io/kubectl/pkg/cmd/drain/drain_test.go

```diff
@@ -609,6 +609,26 @@ func TestDrain(t *testing.T) {
 			expectFatal:  false,
 			expectDelete: false,
 		},
+		{
+			description:  "DS-managed pod with --ignore-daemonsets and --delete-daemonsets",
+			node:         node,
+			expected:     cordonedNode,
+			pods:         []corev1.Pod{dsPod},
+			rcs:          []corev1.ReplicationController{rc},
+			args:         []string{"node", "--ignore-daemonsets", "--delete-daemonsets"},
+			expectFatal:  true,
```

Comment:
While this throws an error, the buffer that we read from in other tests is empty.  To check for the precise error here would require a sizeable refactor for not a lot of gain.  The command's Complete() method will validate the behavior (causes the desired failure in this test), as well as the drain filter inside the actual drain logic (will receive it's own test in a follow-on patch after https://github.com/kubernetes/kubernetes/pull/88337 merges since it has a small refactor to allow testing of filters easier).